\documentclass[10pt,letterpaper]{article}
\usepackage[top=0.85in,left=2.75in,footskip=0.75in]{geometry}

% Use adjustwidth environment to exceed column width (see example table in text)
\usepackage{changepage}

% Use Unicode characters when possible
\usepackage[utf8]{inputenc}

% textcomp package and marvosym package for additional characters
\usepackage{textcomp,marvosym}

% fixltx2e package for \textsubscript
\usepackage{fixltx2e}

% amsmath and amssymb packages, useful for mathematical formulas and symbols
\usepackage{amsmath,amssymb}

% cite package, to clean up citations in the main text. Do not remove.
\usepackage{cite}

% Use nameref to cite supporting information files (see Supporting Information section for more info)
\usepackage{nameref,hyperref}

% line numbers
\usepackage[right]{lineno}

% ligatures disabled
\usepackage{microtype}
\DisableLigatures[f]{encoding = *, family = * }

% rotating package for sideways tables
\usepackage{rotating}

% Remove comment for double spacing
%\usepackage{setspace}
%\doublespacing

% Text layout
\raggedright
\setlength{\parindent}{0.5cm}
\textwidth 5.25in
\textheight 8.75in

% Bold the 'Figure #' in the caption and separate it from the title/caption with a period
% Captions will be left justified
\usepackage[aboveskip=1pt,labelfont=bf,labelsep=period,justification=raggedright,singlelinecheck=off]{caption}

% Use the PLoS provided BiBTeX style
\bibliographystyle{plos2015}

% Remove brackets from numbering in List of References
\makeatletter
\renewcommand{\@biblabel}[1]{\quad#1.}
\makeatother

% Leave date blank
\date{}

% Header and Footer with logo
\usepackage{lastpage,fancyhdr,graphicx}
\usepackage{epstopdf}
\pagestyle{myheadings}
\pagestyle{fancy}
\fancyhf{}
\lhead{\includegraphics[width=2.0in]{PLOS-submission.eps}}
\rfoot{\thepage/\pageref{LastPage}}
\renewcommand{\footrule}{\hrule height 2pt \vspace{2mm}}
\fancyheadoffset[L]{2.25in}
\fancyfootoffset[L]{2.25in}
\lfoot{\sf PLOS}

%% Include all macros below

\newcommand{\lorem}{{\bf LOREM}}
\newcommand{\ipsum}{{\bf IPSUM}}

%% END MACROS SECTION



\usepackage{latexsym}
\usepackage{amsfonts}
\usepackage{pgfplotstable}
\usepackage{url}
\usepackage[nomessages]{fp}
\newcommand{\hConds}{$20$}
\newcommand{\hCondsLBnum}{22}
\newcommand{\hCondsLB}{$\hCondsLBnum$}
\newcommand{\voConds}{$5$}
\newcommand{\vnCondsnum}{23}
\newcommand{\vnConds}{$\vnCondsnum$}
\newcommand{\vConds}{\vnConds{}}
\FPeval{\totConds}{clip(\hCondsLBnum+\vnCondsnum+2)}
\FPeval{\tothConds}{clip(\hCondsLBnum+2)}
\newcommand{\hTotal}{$1442$}
\newcommand{\hGlobal}{$473$}
\newcommand{\hGlobalLB}{$352$}
\newcommand{\hGlobalShuff}{$12$}

\newcommand{\hHorizIntercept}{$-0.36$}
\newcommand{\hHalflife}{$1.9$}

\newcommand{\vnTotal}{$1142$}
\newcommand{\voTotal}{$919$}
\newcommand{\vTotal}{\vnTotal{}}

\newcommand{\voGlobalShuff}{$152$}
\newcommand{\vnGlobalShuff}{$5$}
\newcommand{\vGlobalShuff}{\vnGlobalShuff{}}

\newcommand{\vnGlobal}{$305$}
\newcommand{\voGlobal}{$378$}
\newcommand{\vGlobal}{\vnGlobal{}}

\newcommand{\vnIntercept}{$-0.54$}
\newcommand{\voIntercept}{$-0.54$}
\newcommand{\vHalflife}{$1.29$}
\newcommand{\hProtsInRange}{$414$}
\newcommand{\vProtsInRange}{$221$}

\newcommand{\hMeanSlope}{$1.32$}
\newcommand{\hMeanSlopeStdDev}{$0.57$}
\newcommand{\hSlopeStdErrMean}{$0.34$}
\newcommand{\hRibsMeanSlope}{$1.41$}
\newcommand{\hRibsSlopeStdDev}{$0.32$}
\newcommand{\hRibsSlopeStdErrMean}{$0.23$}
\newcommand{\hGlobalSumSlope}{$1.24$}
\newcommand{\hGlobalSumRsq}{0.91}
\newcommand{\hRibsSumSlope}{$1.37$}
\newcommand{\hRibsSumRsq}{0.89}

\newcommand{\hRibs}{$53$}
\newcommand{\hCorrRibs}{$47$}

\newcommand{\vnRibs}{$53$}
\newcommand{\voRibs}{$54$}
\newcommand{\vRibs}{\vnRibs{}}
\newcommand{\vnCorrRibs}{$52$}
\newcommand{\voCorrRibs}{$52$}
\newcommand{\vCorrRibs}{\vnCorrRibs{}}

\newcommand{\vnMeanSlope}{$1.06$}
\newcommand{\voMeanSlope}{$1.39$}
\newcommand{\vMeanSlope}{\vnMeanSlope{}}

\newcommand{\vMeanSlopeStdDev}{$0.60$}
\newcommand{\vSlopeStdErrMean}{$0.14$}
\newcommand{\vRibsMeanSlope}{$1.48$}
\newcommand{\vRibsSlopeStdDev}{$0.08$}
\newcommand{\vRibsSlopeStdErrMean}{$0.06$}

\newcommand{\vnGlobalSumSlope}{$1.0$}
\newcommand{\voGlobalSumSlope}{$1.39$}
\newcommand{\vGlobalSumSlope}{\vnGlobalSumSlope{}}

\newcommand{\vnGlobalSumRsq}{0.97}
\newcommand{\voGlobalSumRsq}{1.0}
\newcommand{\vGlobalSumRsq}{\vnGlobalSumRsq{}}

\newcommand{\vnRibsSumSlope}{$1.49$}
\newcommand{\voRibsSumSlope}{$1.49$}
\newcommand{\vRibsSumSlope}{\vnRibsSumSlope{}}

\newcommand{\vnRibsSumRsq}{0.97}
\newcommand{\voRibsSumRsq}{0.97}
\newcommand{\vRibsSumRsq}{\vnRibsSumRsq{}}

\newcommand{\hMaxExpVar}{0.082}
\newcommand{\vnMaxExpVar}{0.035}
\newcommand{\voMaxExpVar}{0.04}
\newcommand{\vMaxExpVar}{\vnMaxExpVar{}}

\begin{document}
\vspace*{0.35in}

% Title must be 250 characters or less.
% Please capitalize all terms in the title except conjunctions, prepositions, and articles.
\begin{flushleft}
{\Large
\textbf\newline{A Minimalistic Resource Allocation Model To Explain Ubiquitous Increase In Protein Expression With Growth Rate}
}
\newline
% Insert author names, affiliations and corresponding author email (do not include titles, positions, or degrees).
\\
Uri Barenholz\textsuperscript{1},
Leeat Keren\textsuperscript{2},
Eran Segal\textsuperscript{2},
Ron Milo\textsuperscript{1,*}
\\
\bigskip
\bf{1} Department of Plant and Environmental Sciences, Weizmann Institute of Science, Rehovot, Israel
\\
\bf{2} Department of Computer Science and Applied Mathematics, Weizmann Institute of Science, Rehovot, Israel
\\
\bigskip

* ron.milo@weizmann.ac.il

\end{flushleft}

\linenumbers
\section*{Abstract}
Most proteins show changes in level across growth conditions.
Many of these changes seem to be coordinated with the specific growth rate rather than the growth environment or the protein function.
Although cellular growth rates, gene expression levels and gene regulation have been at the center of biological research for decades, there are only a few models giving a base line prediction of the dependence of the proteome fraction occupied by a gene with the specific growth rate.
We present a simple model that predicts a widely coordinated increase in the fraction of many proteins out of the proteome, proportionally with the growth rate.
The model reveals how passive redistribution of resources, due to active regulation of only a few proteins, can have proteome wide effects that are quantitatively predictable.
Our model provides a potential explanation for why and how such a coordinated response of a large fraction of the proteome to the specific growth rate arises under different environmental conditions.
The simplicity of our model can also be useful by serving as a baseline null hypothesis in the search for active regulation.
We exemplify the usage of the model by analyzing the relationship between growth rate and proteome composition for the model microorganism \emph{E.coli} as reflected in recent proteomics data sets spanning various growth conditions.
We find that the fraction out of the proteome of a large number of proteins, and from different cellular processes, increases proportionally with the growth rate.
Notably, ribosomal proteins, which have been previously reported to increase in fraction with growth rate, are only a small part of this group of proteins.
We suggest that, although the fractions of many proteins change with the growth rate, such changes may be partially driven by a global effect, not necessarily requiring specific cellular control mechanisms.

\section*{Introduction}
A fundamental system biology challenge is to understand and predict changes in gene expression levels.
Early on it was found that the expression of some genes is coordinated with growth rate, rather than with the specific environment.
Classic experiments in bacteria have shown that ribosome concentration increases in proportion to the specific growth rate \cite{Schaechter1958}.
The observed increase in concentration has been interpreted as an increased need for ribosomes at faster growth rates \cite{Neidhardt1999,Dennis2004,Zaslaver2009,Molenaar2009}.
The search for underlying mechanisms in \emph{E.coli} yielded several candidates \cite{Nomura1984} such as the pools of ppGpp and iNTP \cite{Murray_2003,Bosdriesz_2015}, and the tRNA pools through the stringent response \cite{Chatterji2001,Brauer2008}.

In the last two decades, with the ability to measure genome-wide expression levels, it was found that changes in gene expression as a function of growth rate are not limited to ribosomal genes.
In \emph{E.coli}, the expression of catabolic and anabolic genes is coordinated with growth rate, and suggested to be mediated by cAMP \cite{Saldanha2004,You_2013,Peebo_2015}.
In \emph{S.cerevisiae}, it was shown that most of the genome changes its expression levels in response to environmental conditions in a manner strongly correlated with growth rate \cite{Keren2013,Brauer2008,Castrillo2007,Gerosa2013}.
Studies examining the interplay between global and specific modes of regulation, suggested that global factors play a major role in determining the expression levels of genes \cite{Gasch2000,Klumpp2009,Klumpp2014,Scott2010,Scott2011,Scott2014,Berthoumieux2013,Keren2013,Gerosa2013,Valgepea2013,Hui_2015,Weise2015}.

In \emph{E.coli}, this was mechanistically attributed to changes in the pool of RNA polymerase core and sigma factors \cite{Klumpp2008}.
In \emph{S.cerevisiae}, it was suggested that differences in histone modifications around the replication origins \cite{Regenberg2006} or translation rates \cite{Brauer2008} across conditions may underlie the same phenomenon.
Taken together, these studies suggest that the expression of all genes changes with growth rate, with different factors and architectures of regulatory networks yielding differences in the direction and magnitude of these changes \cite{Klumpp2009,Klumpp2014}.

Despite these advancements, many gaps remain in our understanding of the connection between gene expression and growth rate, primarily regarding the underlying mechanisms.
Are there unique factors controlling specific groups of genes, as is suggested by \cite{You_2013,Peebo_2015,Hui_2015,Bosdriesz_2015} and others, or is there a more global phenomenon shared across most genes in the genome?
What fraction of the variability observed in gene expression patterns across different growth conditions results from active adaptation to the specific condition?
To what extent are large clusters of genes regulated by ''master regulator'' factors such as cAMP, and how much by global, gene and condition-independent, response?
Genome-wide proteomic data sets, which take a census of the proteome composition at different growth rates, offer potential insights into these questions and can serve as a basis to explore and compare different models of regulation \cite{Valgepea2013,Schmidt2015,Hui_2015,Peebo_2015}.

Here we present a parsimonious model that quantitatively predicts the relationship between protein abundance and specific growth rate in the absence of gene-specific changes in regulation.
Our model provides a baseline for the behavior of genes in conditions between which they are not differentially regulated, without the need for condition-specific parameters.
The model predicts an increase in protein expression with specific growth rate as an emerging property that is the result of passive redistribution of resources, without need for specific regulation mechanisms such as those detailed above.
On top of this baseline model, different regulatory aspects, that are definitely at play, can be added.
We tested the model against recently published proteomic data sets of \emph{E.coli} spanning different growth conditions \cite{Valgepea2013,Peebo_2015,Schmidt2015,Hui_2015}.
We find a coordinated, positive correlation between the specific growth rate and the fraction of many proteins, from diverse functional groups, out of the proteome.
Although this response accounts for a relatively small part of the total variability of the proteome it is highly relevant for understanding proteome wide studies, as it describes the behavior of about $50\%$ of the proteome genes.
The well-studied ribosomal proteins are found to be a small subset of this group of proteins that increase their fraction with the specific growth rate.
Our analysis suggests that, even if changes in the proteome composition are complex, for a large number of proteins and under many conditions such changes take the form of a linear, coordinated, increase with growth rate.
An increase that can result from cellular resources being freed by down-regulated proteins.
The well studied scaling of ribosome concentration with growth rate can be considered one manifestation of this more general phenomena.
While we present no proof that the mechanism causing the increase in concentration of many proteins with the growth rate is indeed the passive redistribution of bio-synthetic resources, we find the simplicity of this explanation to be a strong evidence for it being at least a partial factor at play.

\section*{Results}

\subsection*{Simple considerations predict passively driven increase in the fraction of proteins as a function of the specific growth rate}
What is the simplest way to model the differences in the proteome composition of two populations of cells, one growing in a permissive environment, and the other facing a more challenging growth condition?
In an attempt to parsimoniously analyze such differences, we have constructed a minimalistic model that predicts the behavior of non-differentially regulated genes across different growth conditions.
Before presenting the model mathematically, we give a brief intuitive depiction.

The model assumes that under a favorable growth condition, the cell actively down-regulates some proteins that are only needed in harsher conditions, as illustrated in Fig.~\ref{fig:model}.
The down regulation of the lac operon in the presence of glucose is a prominent example for this phenomenon.
As a result of only that specific change, the fraction of all other proteins out of the proteome is increased compared to the harsher (e.g. growth on lactose) condition.
In our baseline model, all the other proteins increase their levels and are expected to show the same relative ratios between each other in all conditions.
Specifically, the levels of the proteins forming the bio-synthetic machinery increase, increasing the ratio of the bio-synthetic machinery to the proteome.
The growth rate is dependent on the amount of protein bio-synthesis a cell performs.
The increase in ratio of bio-synthetic machinery to proteome thus results in an expected increase in the growth rate, as depicted in Fig.~\ref{fig:model}.
In our example of the lac operon, in the presence of glucose, the down regulation of lactose metabolism genes leads to faster growth as more bio-synthetic genes are expressed instead.
Therefore, in our model, the effect of growing in high quality media is the down regulation of unnecessary genes, leading to the freeing up of resources that are then redistributed among the genes that are still expressed in this favorable condition.

\begin{figure}[h]
\begin{center}
%\includegraphics[width=\columnwidth]{../figure1.pdf}
\caption{\label{fig:model}
{\bf A minimalistic model predicts that low expression of condition-dependent genes under a permissive growth environment, compared with a restrictive environment, implies larger fraction of all other proteins out of the proteome.}
With this, the ratio of bio-synthesis genes to the rest of the proteome is higher in permissive environments, resulting in faster growth.
%
}
\end{center}
\end{figure}

\paragraph{The expression level of a protein can be decomposed into gene specific control and global expression machinery availability}

The composition of the proteome can in principle be determined by a large number of parameters.
For example, given that an organism expresses 1000 genes across 10 different growth conditions, one could imagine that controlling the expression pattern of all genes across all conditions will require 10,000 parameters (setting the level of every gene in every condition).
Our model proposes an underlying architecture that drastically reduces this amount of parameters, implying that cells control most of the composition of their proteome through fewer degrees of freedom than might be naively expected.

The model separately considers the resulting fraction of every protein out of the proteome as the product of two control mechanisms:
(A) Protein/gene specific controls which only affect the individual protein under a given condition.
These include the gene associated promoter affinity, 5'-UTRs, ribosomal binding site sequence, as well as the presence of specific transcription/translation factors that react with the relevant gene.
We note that while a given transcription factor may affect many genes, the presence or absence of its relevant binding sequence is gene specific, making this control mechanism gene specific in the context of our analysis.
  While some of these controls (such as the ribosomal binding sites) are static, and therefore condition independent, others are dynamic and will differ across different environmental conditions (such as transcription factors state, for genes that are affected by them).
(B) Global expression control based on the availability of bio-synthetic resources, including RNA polymerase, co-factors, ribosomes, amino-acids etc.
  All of these factors can potentially differ across different environmental conditions and no gene can avoid the consequences of changes in them.

In the model, every gene is given an 'affinity-for-expression' (or 'intrinsic-affinity') score that encapsulates its tendency to attract the bio-synthetic machinery, as was first suggested in \cite{Maaloe1969}.
This gene-specific value can in principle change across conditions but a key feature is that the gene intrinsic affinity tends to have the same value across many conditions.
Often two values are enough across all conditions, an "off" and "on" value.

While the intrinsic affinity is a lumped up parameter, combining effects on transcription, translation and other factors together, it can be formally defined in units of protein molecules produced per unit time under standard growth condition $c$.
The use of intrinsic affinities in our model does not require inferring their absolute values and does not require teasing apart the effect of each of their components.
We compare and take ratios of intrinsic affinities to show how they can be used to understand changes in specific growth rate as a result of expression modulation.
Conceptually, intrinsic affinities resemble standard Gibbs free energies of chemical reactions, they integrate many parameters, are given under some standard condition, represent tendencies of processes to occur, and can be used in different contexts without knowing their exact values.

The notion of intrinsic affinities facilitates the representation of the expression pattern under a given condition.
Assuming each gene has only a finite set of affinities, possibly only one or two (for example, on and off states of the lac operon), the expression pattern is reduced to selecting which, out of the total gene-specific small set of possible affinities, each gene gets under the relevant condition.
Given that the selection of expression level for a given gene is driven by some specific environmental cues (translated to, for example, activation of specific transcription factors), the description can be further reduced to determining what cues are present at each condition.

We denote the affinity of gene $i$ under growth condition $c$ by $w_i(c)$.
To determine the resulting fraction of every protein, our model assumes that the bio-synthetic resources are distributed among the genes according to those affinities.
Therefore, the fraction of a specific protein out of the proteome is equal to the specific affinity of the corresponding gene under the condition, divided by the sum of the affinities of all genes under that same condition, as is stated in Eq.(\ref{eq:concentration-ratio}).
Intuitively, one can think of a competition between the genes and transcripts over the bio-synthetic resources, where each gene/transcript attracts resources according to its intrinsic affinity.
To illustrate: if two genes have the same affinity under some condition, their corresponding proteins will occupy identical fractions out of the proteome.
If gene $A$ has twice the affinity of gene $B$ under a given condition, then the fraction protein  $A$ occupies will be twice as large as the fraction occupied by protein $B$ under that condition, etc.
The division by the sum of the affinities of all genes under the condition normalizes the fractions such that the sum of the fractions of all proteins under the condition will be 1.

This relationship can be simply formulated as follows:
\begin{equation}
  \label{eq:concentration-ratio}
  p_i(c)=\frac{P_i(c)}{P(c)}=\frac{w_i(c)}{\sum_jw_j(c)}
\end{equation}
where $p_i(c)$ denotes the fraction out of the proteome of protein $i$ under condition $c$, $P_i(c)$ denotes the mass of protein $i$ under condition $c$ per cell, $P(c)$ denotes the total mass of proteins per cell under condition $c$, and the sum, $\sum_jw_j(c)$, is taken over the intrinsic affinities of all the genes the cell has.

This equation emphasizes that the observed fraction of a protein is determined by the two factors mentioned above: the specific affinity of the protein/gene, that is present in the numerator, and also, though less intuitive, the affinity of all other genes under the growth condition (affecting the availability of bio-synthetic resources), as reflected by the denominator.

For simplicity, the model refers to the fraction of each specific protein in the proteome and not to the protein concentration.
The corresponding concentration in the biomass can be calculated using the concentration of total protein in the biomass.
In \emph{E.coli}, this concentration is known to slightly decrease in a linear manner with the specific growth rate \cite{Bremer1987,Valgepea2013,Scott2014}.
As the decrease in total proteome mass per cell dry weight is relatively small, it will not cause a qualitative difference in our findings even when considering specific protein concentration and not protein fraction out of the proteome (for further discussion see Methods).


\paragraph{A change in growth condition triggers changes in expression of specific proteins that indirectly affect the whole proteome}

Different environmental conditions require the expression of different genes.
For example, the expression of amino-acids synthesizing enzymes is required only in culture media lacking amino-acids \cite{24656150,10515934}.
Therefore, the cell can infer the presence or absence of amino-acids in the growth media and, regulate the affinities of the synthesizing genes accordingly.
If we consider a gene $i$, whose specific affinity is not dependent on the presence of amino-acids, we suggest that its fraction will still change between the two conditions as the affinities of other condition specific genes change, thereby redirecting the bio-synthetic capacity.
In mathematical terms this will change the denominator in Eq.(\ref{eq:concentration-ratio}) and thus affect the distribution of resources between all of the expressed genes.


Generalizing this notion, we can divide the proteins into those whose intrinsic affinity remains constant across all of the considered conditions, and those whose intrinsic affinity changes between at least some of the conditions (Fig.~\ref{fig:model}).
An interesting consequence is that proteins whose intrinsic affinities remain constant also maintain their relative ratios across these conditions with respect to each other, as observed experimentally in \emph{S.cerevisiae} in \cite{Keren2013}.

\paragraph{Growth rate is the outcome of proteome composition which is dictated in turn by the environmental conditions}

While it is sometimes implied that different cellular components are regulated by the growth rate, our model considers the growth rate as an outcome of the environmental conditions that affect the proteome composition.
Specifically, we assume that the doubling time is proportional to the ratio of the total amount of proteins per cell and the proteins involved in bio-synthesis in that cell.
The larger the ratio of total proteins to bio-synthesis proteins is, the longer these bio-synthesis proteins will need to duplicate the proteome, resulting in a longer doubling time.

To illustrate this assumption concretely, one could think about the synthesis of polypeptides.
If a cell has $R$ actively translating ribosomes, each of which synthesizing polypeptides at a rate of $\eta \approx 20$ amino acids per second, the bio-synthetic capacity of the cell will be limited to $\approx \eta R$ amino acids per second.
If the total amount of protein in that same cell is $P$ (measured in amino acids count), it follows that the time it will take the actively translating ribosomes to synthesize the proteins for an identical daughter cell is $\tau \approx \frac{P}{\eta R}$ (up to a $\ln(2)$ factor resulting from the fact that the ribosomes also synthesize more ribosomes during the replication process and that these new ribosomes will increase the total rate of polypeptides synthesis).


The theoretical lower limit of the doubling time, $T_B$, will be achieved when all of the proteome of the cell is the bio-synthetic machinery.
If the bio-synthetic machinery is only half of the proteome, the doubling time will be $2T_B$ etc.

To integrate the notion of total protein to bio-synthetic protein ratio into our model, we make the following simplifying assumption:
There is a group of bio-synthetic genes (e.g. genes of the transcriptional and translational machineries) the affinities of which remain constant across different growth conditions, that is, these genes are not actively differentially regulated across different conditions.
Furthermore, we assume that the machineries these genes are involved in, operate at relatively constant rates and active to non-active ratios across conditions \cite{Bremer1987}.
We are aware that these values are estimated to change by up to 2 fold and, for now, consider such changes to be negligible.
We incorporate the effects of changes to synthesis rates into our model below.

Formally, we define the group of bio-synthesis genes, $G_B$, such that, for every gene that belongs to this group, $k \in G_B$, its affinity, $w_k(c)$ is constant regardless of the condition, $c$.
\begin{equation}
  \label{eq:biosynth-def}
  w_k(c)=w_k
\end{equation}

To keep our notations short, we will define the condition independent sum over all of these bio-synthesis genes as the constant:
\[
W_B = \sum_{k\in G_B}w_k
\]

The doubling time under a given condition, $\tau(c)$, will be proportional to the ratio of total protein to bio-synthesis protein under that condition, with the proportionality constant $T_B$:
\begin{equation}
  \label{eq:gr-ratio}
  \tau(c) = T_B\frac{P(c)}{\sum_{k\in G_B}P_k(c)}=T_B\frac{\sum_jw_j(c)}{W_B}
\end{equation}
Therefore, the model reproduces an increase in the doubling time for conditions requiring larger amounts of non-bio-synthetic proteins (i.e. higher values in the sum across $w_j$).

\paragraph{The fraction of a non-differentially regulated protein is expected to increase with the growth rate}

Recalling that the connection between the growth rate and the doubling time is: $g(c)=\frac{\ln(2)}{\tau(c)}$, we now combine Eq.(\ref{eq:concentration-ratio}) with Eq.(\ref{eq:gr-ratio}) to get a prediction for the single protein fractions $p_i$:
\begin{equation}
  \label{eq:default-response}
  p_i(c)=\frac{w_i(c)}{\sum_jw_j(c)}=\frac{w_i(c)}{W_B}\frac{W_B}{\sum_jw_j(c)}=\frac{w_i(c)}{W_B}\frac{T_B}{\ln(2)}g(c)
\end{equation}

By incorporating all the condition-independent constants ($W_B$, $T_B$, $\ln(2)$) into one term, $A$, we can simplify to:
\begin{equation}
  \label{eq:final-conc}
  p_i(c)=Aw_i(c)g(c)
\end{equation}
Hence, for every two conditions between which gene $i$ maintains its affinity, ($w_i(c_1)=w_i(c_2)$), the fraction $p_i(c)$ protein $i$ occupies in the proteome scales in the same way as the growth rate ($g(c)$) between these two conditions.

To summarize, the simplified model we have constructed predicts that, under no specific regulation, the fraction a non-regulated protein occupies out of the proteome should scale proportionally with the growth rate.
A group of such proteins would therefore maintain their relative ratios across conditions.

\paragraph{Protein degradation differentiates between measured growth rate and biomass synthesis rate}

In the following two sections we analyze the effects of expanding our model to account for two biological effects: protein degradation and changes in the rates at which molecular machines operate.

The model we developed predicts that when the growth rate approaches zero, the fraction of every protein with constant affinity also approaches zero.
This approach to zero applies specifically to the bio-synthesis genes, that have constant affinities according to our assumptions.
However, it is known that the fractions of these proteins, and specifically of ribosomal proteins does not drop to zero when the growth rate approaches zero \cite{ingraham1983growth,Pedersen1978a}.
We can account for this phenomenon by including protein degradation in our model.

We assume the degradation rate to be constant for all genes and conditions.
Clearly the biological situation is much more complicated.
We note that with the future availability of detailed information on degradation rates, it will be straightforward to extend the model.
Experimental evidence in \emph{S.cerevisiae} suggest that protein degradation occurs at relatively slow rates with half lives of $\approx10$ hours \cite{Christiano2014}.
The \emph{observed} growth rate, $g$, is the amount of proteins produced \emph{minus} the amount of proteins degraded.
To illustrate, at zero growth rate, the implication is not that no proteins are produced, but rather that proteins are produced at exactly the same rate as they are degraded.

Integrating this notion into the model means that the bio-synthesis capacity needs to suffice to re-synthesize all the degraded proteins.
Hence, where the equations previously referred to the cellular growth rate, $g$, as the indicator of protein synthesis rate, they should in fact refer to the cellular growth rate plus the degradation rate, as that is the actual rate of protein synthesis.
If we denote by $\alpha$ the degradation rate, Eq.(\ref{eq:final-conc}) should thus be rewritten as:
\begin{equation}
  \label{eq:final-conc-deg}
  p_i(c)=Aw_i(c)(g(c)+\alpha)
\end{equation}
This equation predicts linear dependence of the fraction of unregulated proteins on the growth rate, with an intercept with the horizontal axis occurring at minus the degradation rate (\nameref{fig:theoreticalpred}).
Thus, at zero growth rate, the fraction of non-differentially regulated proteins out of the proteome is positive, equaling $Aw_i(c)\alpha$.

\paragraph{Slower rates of biological processes in lower growth rates affect the relationship between proteome composition and growth rate}

The simplified model assumes that the doubling time is proportional to the ratio of total protein to bio-synthetic protein.
This assumption fails if the rate at which each bio-synthetic machine operates changes across conditions.
While ribosomal translation rates and mRNA transcription rates are relatively constant per synthesizing machinery unit, they may change up to 2 fold across different conditions \cite{Bremer1987}.
Replacing this assumption by an interdependence of bio-synthesis rate with growth rate (such that, the faster the growth, the faster the synthesis rates, per machine)\cite{Bremer1987,Valgepea2013}, will affect the resulting predictions as well.
This effect is formally analyzed in \nameref{theoreticalintercep}.
Slower bio-synthesis rates under slower growth rates imply that, compared with the model prediction, higher fraction of bio-synthesis proteins is needed to achieve a given growth rate.
Thus, lower synthesis rates under slower growth rates will be reflected by a lower slope and higher interception point for non-regulated proteins than those predicted by the constant-rate version of the model, as is depicted in \nameref{fig:theoreticalpred}.

To summarize, our theoretical model predicts that the default behavior of non-differentially regulated proteins between two conditions is to maintain a fraction that is proportional to the growth rate.
The faster the growth rate, the higher the fraction.
Such proteins should maintain their relative concentrations w.r.t. each other.
Degradation and changes in rates of molecular machineries at slow growth result in predicting non-zero fraction for such proteins even when the growth rate is zero, resulting in a more moderate response of the fraction to the growth rate.

\subsection*{Analysis of proteomic data sets}
Our theoretical model predicts that the fraction of many proteins proportionally and coordinately scales up with the specific growth rate across different growth conditions.
To assess the extent to which this prediction is reflected in actual proteome compositions, we present analysis of two published proteomics data sets of \emph{E.coli}, \cite{Peebo_2015} and \cite{Schmidt2015}.
These data sets use mass spectrometry to evaluate the proteomic composition of \emph{E.coli} under \vConds{} different growth rates using an accelerostat \cite{Paalme_1995}, and \hConds{} different growth conditions, spanning both different carbon sources and chemostat-controlled growth rates, respectively.
The data set from \cite{Schmidt2015} contains more conditions than those analyzed below, see Methods for further details.

Other proteomic data sets of \emph{E.coli} have been published in recent years (\cite{Valgepea2013} and \cite{Hui_2015}).
These data sets were measured under conditions that make them less adequate for our analysis as is further discussed in \nameref{furtherdatasetanalysis}.

We note that continuously controlled growth rates (such as those obtained in chemostats and accelerostats) tend to yield uniform results that roughly split the proteins to two groups, a group of proteins the fraction of which increases with the growth rate, and a group of proteins the fraction of which decreases with growth rate (\nameref{fig:grcorr_dbs}).
Such sets of conditions are generated by modifying the growth rate using a single, continuous external parameter, such as the carbon source concentration.
Therefore, a relatively fixed set of genes is expected to be actively regulated (up or down) across all of the conditions.
As a result, such conditions are less adequate for the purpose of identifying differences between actively regulated genes and passively expressed genes as they do not readily allow distinction between these two groups.

While we suggest that control states are discrete and finite, it is not a necessary condition for our mathematical analysis to hold, and allowing for continuous modulation of affinities of genes does not affect the model prediction of increased fraction with growth rate for un-modulated affinities.
We therefore include continuous growth conditions together with the distinct conditions in our analysis, in the data sets that involve more than a single growth rate controlling factor.

\paragraph{A large fraction of the proteome is positively correlated with growth rate}

Our model predicts that proteins that are not differentially regulated between conditions should increase in fraction with the growth rate.
The identification presented here of a large number of proteins, from unrelated functional groups and with unknown common regulating mechanism, that increase in fraction with the growth rate, may serve as an indication that a passive, global mechanism, such as that proposed by our model, is at play.
To test this prediction, we calculated the Pearson correlation of every protein with the growth rate (Fig.~\ref{fig:growthcorr}, panels A and B).
We find that about a third of the proteins (\hGlobal{} out of \hTotal{} measured in the data set from \cite{Schmidt2015}, and \vGlobal{} out of \vTotal{} in the data set from \cite{Peebo_2015}) have a strong positive ($>0.5$, see also \nameref{corrthreshold}) correlation with the growth rate.
These values are much higher than those obtained for randomized data sets (\hGlobalShuff{} and \vGlobalShuff{} strongly positively correlated proteins for the two data sets, respectively, as is further discussed below and is seen in Fig.~\ref{fig:growthcorr}C).
Assessing the agreement between the two data sets by comparing the correlation with the growth rate of every protein across the 4 glucose limited chemostat growth conditions in \cite{Schmidt2015} and the 9 glucose limited accelerostat conditions in \cite{Peebo_2015} gives a moderate covariance of $\approx 0.4$ (\nameref{dbscorr} and \nameref{fig:chemcorrcomp}).
Strong negative correlation with growth rate is much less common in the data set from \cite{Schmidt2015}.
It is common in the data set from \cite{Peebo_2015}, where we speculate that it results from the specific way by which growth rate was controlled, namely by implicitly controlling nutrient concentration via an accelerostat, as was discussed above.
Notably, in both data sets, the proteins that have a high correlation with the growth rate are involved in many and varied cellular functions and span different functional groups (See \nameref{tab:corrbreakdownh} and \nameref{tab:corrbreakdownv}).


Previous studies already found that ribosomal proteins are strongly positively correlated with growth rate \cite{Pedersen1978a, ingraham1983growth, Klumpp2008}.
Our analysis agrees with these findings as we find the fraction of the vast majority of the ribosomal proteins to be strongly positively correlated with growth rate (\hCorrRibs{} out of \hRibs{} in the data set from \cite{Schmidt2015} and \vCorrRibs{} out of \vRibs{} in the data set from \cite{Peebo_2015}).
However, we also find that the group of proteins strongly positively correlated with growth rate reach far beyond the previously discussed group of ribosomal proteins (\nameref{tab:corrbreakdownh} and \nameref{tab:corrbreakdownv}).
Importantly, the proteins that we find to be strongly positively correlated with growth rate are not generally expected to be co-regulated, and their behavior does not seem to be the result of any known transcription factor or regulation cluster response \cite{23203884}.

\begin{figure}[h]
\begin{center}
%\includegraphics[width=1\columnwidth]{../GrowthRateCorrelation.pdf}
\caption{\label{fig:growthcorr}
{\bf A strong positive Pearson correlation between the fraction out of the proteome and the growth rate is observed for a large number of proteins in two data sets.}
(A-B) Shown are histograms displaying the correlations of all proteins to growth rate in the data from \cite{Schmidt2015} (A) and \cite{Peebo_2015} (B).
Functional protein groups are denoted by different colors.
Thresholds defining high correlation are marked in dashed lines and further discussed in \nameref{corrthreshold}.
(C) Shuffling the amounts of every protein across conditions for the data set from \cite{Peebo_2015} reveals the bias towards positive correlation with growth rate is non-trivial.
%
}
\end{center}
\end{figure}

\paragraph{Proteins positively correlated with growth rate share a similar response}

Our model predicts that non-differentially regulated proteins should preserve their relative ratios across conditions.
We refer to such proteins as being \emph{coordinated} or \emph{coordinately expressed}.
We have shown above that many proteins are positively correlated with growth rate.
However, we note that having similar correlation with growth rate for different proteins does not imply that such proteins are coordinated, i.e. that they share the same scaling with growth rate.
Theoretically, proteins with identical correlation with growth rate may have very different slopes or fold changes with increasing growth rate.

In order to examine how similar the behavior with growth rate is for the group of strongly positively correlated proteins, we normalized each of them to its mean abundance (see Methods) and calculated the slope of a linear regression line for the normalized fraction vs. the growth rate (Fig.~\ref{fig:globalfit}).
This normalization procedure elucidates how the amount of each protein scales across different conditions, relative to its mean fraction out of the proteome.
The slopes of $\approx \frac{2}{3}$ of the proteins lie in the range $(0.5,2)$ with the highest slopes being $\approx 5$.
A slope of $0.5$ means that the fraction of the protein changes by $\pm12\%$ around its average fraction in the range of growth rates measured, whereas a slope of $2$ indicates a change of $\pm50\%$.
Hence, the relative amounts of proteins with slopes in the range of $(0.5,2)$ change by at most just over 2-fold over the range of growth rates measured.

To understand whether the observed distribution is coordinated, and can result from the noise levels present in the data, we calculated, for every protein, the standard error with respect to the regression line that best fits its fractions.
Given these standard errors we generated the expected distribution of slopes that would result by conducting our analysis on proteins that share a single, identical slope, but with the calculated noise in measurement.
The expected distribution is shown in gray line in Fig.~\ref{fig:globalfit} (Further details on the calculation as well as the deviation in maximum between the expected and observed distributions are discussed in \nameref{slopedistexp}).
The two data sets show different characteristics of the expected distribution.
While the expected distribution corresponding to the data set from \cite{Schmidt2015} coincides with the observed variability in calculated slopes, supporting the notion of a coordinated response, for the data from \cite{Peebo_2015} the expected distribution is much narrower, suggesting a bi-modal distribution.
Future studies may uncover the factors underlying the difference between the distributions of the two data sets.

Next we examined how the response of the strongly correlated proteins relates to the well-studied response of ribosomes concentration.
To that end, we performed the same analysis of slopes, restricting it to ribosomal proteins alone, as is shown by the stacked green bars in Fig.~\ref{fig:globalfit}.
We find that strongly correlated proteins and ribosomal proteins scale in similar ways (slope of \hRibsSumSlope{} with $R^2=\hRibsSumRsq{}$ for the sum of ribosomal proteins vs. \hGlobalSumSlope{} and $R^2=\hGlobalSumRsq$ for the sum of all strongly correlated proteins, in the data from \cite{Schmidt2015}, and slope of \vRibsSumSlope{} with $R^2=\vRibsSumRsq{}$ for ribosomal proteins vs. \vGlobalSumSlope{} and $R^2=\vGlobalSumRsq{}$ for all strongly correlated proteins, in the data from \cite{Peebo_2015}. See also \nameref{fig:ribsnonribs}), implying that the observed response of ribosomal proteins to growth rate is not unique and is coordinated with a much larger fraction of the proteome, thus encompassing many more cellular components.

Our results support the notion that a large number of proteins maintain their relative concentrations across different growth conditions and thus extend the scope of similar results obtained for \emph{S.cerevisiae} in \cite{Keren2013} and for expression levels in \emph{E.coli} under stress conditions \cite{Kaneko2014}.
In contrast to other approaches, our model suggests a mechanism for this coordinated expression changes that is not based on shared transcription factors but rather is a result of passive redistribution of resources.

\begin{figure}[h]
\begin{center}
%\includegraphics[width=1\columnwidth]{../AllProtsVSRibosomalNormalizedSlopes.pdf}
\caption{\label{fig:globalfit}
{\bf Histogram of the slopes of regression lines for the highly correlated with growth proteins (\hGlobal{} and \vGlobal{} proteins in the left and right panels respectively).}
    Ribosomal proteins are stacked in green on top of the non ribosomal proteins, marked in blue.
    Proteins fractions were normalized to account for differences in slopes resulting from differing average fractions (Methods).
    The expected distribution of slopes given the individual deviations of every protein from a linear regression line, assuming all proteins are coordinated, is plotted in gray.
    Dashed vertical lines at $0.5$ and $2$ represent the range at which the slopes of $\approx \frac{2}{3}$ of the proteins lie.
    Left panel - data from \cite{Schmidt2015}, right panel - data from \cite{Peebo_2015}.
    High correlation proteins share similar normalized slopes, implying they are coordinated, maintaining their relative ratios across conditions (see text for further details).
    Ribosomal proteins, shown in green, scale with growth rate in a manner similar to the rest of the high correlation proteins (see text and \nameref{fig:ribsnonribs}).
%
}
\end{center}
\end{figure}

\paragraph{Changes in the proteome across environmental conditions are dominated by proteins that are positively correlated with growth rate}

To assess the significance of the positive correlation of proteins with growth rate, out of the total change in proteome composition across conditions, we summed the fractions of all of the proteins that are strongly correlated with growth rate across the conditions measured and plotted their total fraction against the growth rate in Fig.~\ref{fig:globalgrcorr}.
Both data sets show that the fraction of these proteins change $\approx 2$ fold across a $\approx 5$ fold change in the growth rate under the different growth conditions.
This change is smaller than the $1:1$ change predicted by our basic model and the deviations may partly result from the effects of degradation and varying bio-synthesis rates, as is discussed above.
Most of the variability of the total fraction of these proteins can be explained by the growth rate ($R^2$ of $\hGlobalSumRsq$ in the data set from \cite{Schmidt2015} and $\vGlobalSumRsq$ in the data set from \cite{Peebo_2015}).
Importantly, the strongly correlated proteins form a large fraction of the proteome, exceeding $50\%$ of the proteome by weight, at the higher growth rates.
This is a much higher fraction than the one obtained for randomized data sets ($<4\%$, as is further discussed below)
Thus, when considering the changes in proteome composition across conditions, we find that, at higher growth rates, more than $50\%$ of the proteome composition is affected by the coordinated response of the same group of proteins with growth rate.
In the extreme case that all of the change in the expression of the strongly positively correlated with growth rate proteins results from passive redistribution of resources, this change implies that, across the conditions used, $\approx 25\%$ of the proteome by mass changed from being dedicated to condition specific proteins, to being dedicated to the $\approx 1/3$ of the proteins that are strongly positively correlated with growth rate.

Despite the magnitude of this phenomena, the fraction of the total variability in the proteome that is accounted for by this linear response is only $\approx 8\%$  in the data set from \cite{Schmidt2015} and even lower in \cite{Peebo_2015} (\nameref{fig:threshold}).
While this fraction is low, it is still much higher than the equivalent $2\%$ obtained for a randomized data set based on the data from \cite{Schmidt2015}, as is described below.
This relatively low explained variability fraction is primarily the result of two factors: the linear response applies only to $<0.4$ of the proteins, leaving the rest of the proteins with no prediction, and experimental noise in whole proteome measurement techniques, estimated at $\approx20\%$.
Further discussion of the fraction of variability explained can be found in \nameref{corrthreshold}.

\begin{figure}[h]
\begin{center}
%\includegraphics[width=1\columnwidth]{../GlobalClusterGRFit.pdf}
\caption{\label{fig:globalgrcorr}
{\bf Fraction of the proteome occupied by proteins that are strongly positively correlated with growth rate.}
The accumulated sum of the proteins that are strongly positively correlated with growth rate (defined as having a correlation above $0.5$), as a fraction out of the proteome, with linear regression lines is shown.
These proteins form a large fraction ($\ge 50\%$) out of the proteome at higher growth rates.
The accumulated fraction of the strongly correlated proteins doubles as the growth rate changes by about 5-fold.
Assuming constant degradation rates, the trend lines correspond to protein half life times of $\approx 1.7$ hours.
Randomized data sets result in much fewer strongly positively correlated with growth rate proteins, implying a much smaller accumulated fraction (hollow circles).
%
}
\end{center}
\end{figure}

\paragraph{The statistical features we find do not naturally rise in randomized data sets}

We performed two tests to verify that the trends we find, namely, the large fraction of proteins with a strong correlation with growth rate, the coordination among these proteins, their large accumulated fraction out of the proteome, and the fraction of variability explained by a single linear regression approximation of their fractions, are all non-trivial characteristics of the data set that do not naturally rise in randomly generated data but that do arise if our model is correct.
To this extent we repeated our analysis on two simulated data sets:
\begin{itemize}
\item A data set at which the amount of every protein was shuffled across the different conditions.
\item A synthetic, simulated data set, based on the conditions and growth rates of the data set from \cite{Schmidt2015}, assuming half the proteins being perfectly coordinated and linearly dependent on growth rate, with parameters similar to those found in our analysis, and the other half having no correlation with growth rate, and with a simulated normally distributed measurement noise of $25\%$.
\end{itemize}

We find that in the shuffled sets the number of proteins being significantly positively correlated with
growth rate is much smaller than found in the real data sets (\hGlobalShuff{} vs. \hGlobal{} in the data set from \cite{Schmidt2015} and \vGlobalShuff{} vs. \vGlobal{} in the data set from \cite{Peebo_2015}) as is shown in Fig.~\ref{fig:growthcorr}C.
As a consequence, these proteins now occupy a much smaller fraction out of the proteome mass-wise ($<4\%$ on average across conditions vs. $\approx 40\%$ in the real data sets) as is shown in Fig.~\ref{fig:globalgrcorr}.
Finally, the fraction of variability in the proteome that can be explained by a single linear regression to these proteins is smaller for the shuffled data sets than that obtained for the real data set ($2\%$ vs. $8\%$ for a threshold of $R\ge0.5$ for the data from \cite{Schmidt2015} and $1\%$ vs. $3.5\%$ for the data set from \cite{Peebo_2015}), as is seen in \nameref{fig:shuffledexpvar}.

We find that the simulated (second) set does display similar characteristics to those we find in the real data, confirming that if, indeed, our model is valid, experimental measurements would overlap with those that we obtained as is seen in \nameref{fig:simulated}.

\section*{Discussion}
We presented a parsimonious model connecting the fraction of proteins out of the proteome and the growth rate as an outcome of the limited bio-synthesis resources of cells.
The notion of intrinsic affinity for expression, first presented in \cite{Maaloe1969}, and rarely used ever since, was re-introduced as a key determinant for the differences in expression of different proteins under a given growth condition.
The integration of the notion of intrinsic affinity for expression with the limited bio-synthesis capacity of cells was shown to result in a simple mechanism predicting increased fraction of many proteins with the growth rate, without assuming regulation by specific transcription factors for these proteins.

The framework we present emphasizes the importance of accounting for global factors, that are reflected in the growth rate, when analyzing gene expression and proteomics data, as was noted before \cite{Maaloe1969,Brauer2008,Klumpp2009,Klumpp2014,Scott2011,Scott2014,Scott2010,Berthoumieux2013,Keren2013,Gerosa2013,Valgepea2013,Hui_2015,Peebo_2015,Weise2015}.
Specifically, we suggest that the default response of a protein (that is, the change in the observed expression of a protein, given that no specific regulation was applied to it) is to linearly increase with growth rate.
We point out that, as non-differentially regulated proteins maintain their relative abundances, one can deduce the parameters of the linear increase with growth rate of any non-differentially regulated protein by observing the scaling of other such proteins and fixing the ratio between the protein of interest and the reference proteins.

We analyze two recent whole proteome data sets to explore the scope and validity of our model.
We characterize a coordinated response in \emph{E.coli} between many proteins and the specific growth rate.
This response spans proteins from various functional groups and is not related to the specific medium of growth.
A similar phenomena is observed for \emph{S.cerevisiae} as was reported in \cite{Keren2013} and may thus be conserved across various organisms and domains of life.
Our analysis suggests that, while changes in the proteome composition may seem complex, for a large number of proteins and under many conditions, they can be attributed to a linear, coordinated, increase with growth rate, at the expense of other, down-regulated proteins.
The well studied scaling of ribosomes concentration with growth rate can be considered one manifestation of the more general phenomena we describe here.
We find that this response is not unique to ribosomal proteins but is, in fact, shared with many other proteins spanning different functional groups.
Furthermore, the linear dependence slope and explained variability of fraction levels of proteins explained by linear correlation with growth rate is similar among the ribosomal proteins versus all the proteins with high correlation with the growth rate.

Many studies monitored the ribosome concentration in cells and its interdependence with growth rate \cite{Schaechter1958,Bremer1987,Zaslaver2009,Scott2010,Valgepea2013,Peebo_2015,Hui_2015}(many of them indirectly).
While in all of these studies a linear dependence of ribosome concentration with growth rate was observed, in some cases different slopes and interception points were found to describe this linear dependence, compared with the observations in our study.
A discussion of various reasons that may underlie these differences is given in \nameref{ribosomeconc}.

Interestingly, our model suggests that a linear correlation between ribosomal proteins and the growth rate might be achieved without special control mechanisms.
Nonetheless, many such mechanisms have been shown to exist \cite{Nomura1984,Scott2014} (for a comparison to the models suggested in \cite{Scott2010,Scott2011,Scott2014} and \cite{Hui_2015}  see \nameref{scottcompare}).
We stress that the existence of such mechanisms does not contradict the model.
Mechanisms for ribosomal proteins expression control may still be needed to achieve faster response under changing environmental conditions or a tighter regulation to avoid unnecessary production and reduce translational noise.
Furthermore, such mechanisms may be crucial for synchronizing the amount of rRNA with ribosomal proteins as the two go through different bio-synthesis pathways.
The well known and widely studied ppGpp is one example of a regulatory mechanism that, while affecting the expression of various proteins, seems to be mainly targeted at stable RNA synthesis (rRNA and tRNA) and not at ribosomal proteins \cite{Srivatsan2008}.
Nevertheless, the fact that many non-ribosomal proteins share the same response as ribosomal proteins do, poses interesting questions regarding the scope of such control mechanisms, their necessity and the trade-offs involved in their deployment.

The findings in this study support and broaden the findings in other recent studies.
Specifically, for \emph{S.cerevisiae} a few recent studies found that the concentration of the majority of the proteins is coordinated across conditions and increases with growth rate \cite{Keren2013,Gasch2000,Brauer2008}.
In principle, the model we suggest here can be applied to any exponentially growing population of cells and may thus also serve as a potential explanation for the phenomena observed in these studies and others.

Modeling of cellular metabolism is an advanced field and whole cell models are built in considerable numbers.
Future models can explicitly include the role of passive redistribution according to the lines presented here and thus allow testing the improvement in model prediction as well as model robustness and need for free parameters with versus without taking it into account.
Moreover, passive redistribution of resources can then be compared with active regulation of specific genes to pin-point where such regulation is needed according to the model, and where it is redundant.

Other recently published studies in \emph{E.coli} have suggested different models and in some cases have results and predictions that do not coincide with those presented in this study.
Notably, in \cite{Klumpp2009,Klumpp2014} a decreased protein concentration for unregulated genes is predicted.
Theoretically, the two models differ in one key assumption.
Whereas the models in \cite{Klumpp2009,Scott2010,Scott2011,Klumpp2014} consider the 'growth driving' proteins to be actively up regulated under favorable conditions, our model suggests that the higher concentration of these 'growth driving' proteins can be a byproduct of the down regulation of unnecessary proteins under favorable conditions.
This difference in assumptions leads to differing predictions regarding the response in expression to growth rate under the two models.
The models from \cite{Klumpp2009,Scott2010,Scott2011,Klumpp2014} predict that unregulated proteins will decrease in concentration with the growth rate, whereas our model predicts an increase in their concentration with growth rate.

Other aspects may also play a role in the differing predictions and observations of the two models.
The calculation in \cite{Klumpp2009} relies on data collected under higher growth rates than those available in the data sets analyzed here.
The predictions of the model are based on the deduced dependence of various bio-synthesis process rates and physiological properties of the cells on the growth rate, properties that are, in turn, used to calculate the expected protein concentration for unregulated proteins under the different growth rates.
The model in \cite{Klumpp2009} refers to protein concentration and not to protein fraction out of the proteome. These quantities may differ due to changes in the ratio of total protein mass to cell dry weight, or cell volume, as a function of the growth rate.
Thus, this approach is markedly different than the approach we take, which assumes relatively small changes in bio-synthetic rates as a function of growth rate and focuses on the limited bio-synthesis resources as the main driver of changes in the resulting fraction of proteins out of the proteome.
We find the amount and variety of proteins that display similar response to the 'growth driving' proteins with respect to the growth rate to be a significant, albeit circumstantial, evidence that the assumption our model makes may be valid.
As the model in \cite{Klumpp2009} was only tested against a handful of proteins, further data collection is required to decide which of the two models better describes the global effects of growth rate on proteome composition.

The expected availability of increasing amounts of whole proteome data sets, with higher accuracy levels, will enable further investigation of the details of cellular resource distribution.
With our model serving as a baseline, the analysis of such future data sets will shed more light on the relative roles of carefully tuned response mechanisms versus global, passive effects in shaping the proteome composition under different growth environments.

\section*{Materials and Methods}
\paragraph{Data analysis tools}

All data analysis was performed using custom written software in the Python programming language.
The data analysis source code is available through github at: http://github.com/uriba/proteome-analysis
Analysis was done using SciPy \cite{Oliphant2007}, NumPy \cite{Community2011} and the Pandas data analysis library \cite{McKinney2011}.
Charts where created using the MatPlotLib plotting library \cite{Hunter2007}.

\paragraph{Normalizing protein fractions across conditions}

Our analysis aims at identifying proteins that share similar expression patterns across the different growth conditions.
For example, consider two proteins, $A$ and $B$ measured under two conditions, $c_1$ and $c_2$.
Assume that the measured fractions out of the proteome of these two proteins under the two conditions were $0.001$ and $0.002$ for $A$ under $c_1$ and $c_2$ respectively, and $0.01$ and $0.02$ for $B$ under $c_1$ and $c_2$ respectively.
These two proteins therefore share identical responses across the two conditions, namely, they double their fraction in the proteome in $c_2$ compared with $c_1$.

The normalization procedure scales the data so as to reveal this identity in response.
Dividing the fraction of each protein out of the proteome by the average fraction of that protein across conditions yields the normalized response.
In the example, the average fraction of $A$ across the different conditions is $0.0015$ and the average fraction of $B$ is $0.015$.
Thus, dividing the fraction of every protein by the average fraction across conditions of that same protein yields:
\[
A'_{c_1}=\frac{A_{c_1}}{\bar{A}}=\frac{0.001}{0.0015}=\frac{2}{3}=\frac{0.01}{0.015}=\frac{B_{c_1}}{\bar{B}}=B'_{c_1}
\]
for $c_1$ and:
\[
A'_{c_2}=\frac{A_{c_2}}{\bar{A}}=\frac{0.002}{0.0015}=\frac{4}{3}=\frac{0.02}{0.015}=\frac{B_{c_2}}{\bar{B}}=B'_{c_2}
\]
for $c_2$ showing $A$ and $B$ share identical responses across $c_1$ and $c_2$.

The general normalization procedure thus divides the fraction of protein $i$ under condition $c$, $p_i(c)$ by the average fraction of protein $i$ across all of the conditions in the data set, $\bar{p}_i$, to give the normalized fraction under condition $c$, $p'_i(c)=\frac{p_i(c)}{\bar{p}_i}$.

This normalization procedure has been applied prior to calculating the slopes of the regression lines best describing the change in fraction out of the proteome of every protein as a function of the growth rate.
Furthermore, when analyzing the variability explained by linear regression on the sum of fractions of all proteins presenting a high correlation with the growth rate, the same normalization procedure was made in order to avoid domination by the high abundance of a few proteins in that group.

\paragraph{Calculation of protein concentration}

In this study, we use the mass ratio of a specific protein to the mass of the entire proteome, per cell, as our basic measure for the bio-synthetic resources a specific protein consumes out of the bio-synthetic capacity of the cell.
We find this measure to be the best representation of the meaning of a fraction a protein occupies out of the proteome.
However, we note that if initiation rates are limiting (e.g. if RNA polymerase rather than ribosomes become limiting), and not elongation rates, then using molecule counts ratios (the number of molecules of a specific protein divided by the total number of protein molecules in a cell) rather than mass ratios may be a better metric.
We compared these two metrics and, while they present some differences in the analysis, they do not qualitatively alter the observed results.

There are different, alternative ways to assess the resources consumed by a specific protein out of the resources available in the cell.
On top of the measures listed above, one could consider either the total mass or molecule count of a specific protein out of the biomass, rather than the proteome, or out of the dry weight of the cell, both of which vary with the ratio of total protein to biomass or dry weight which was neglected in our analysis.
Moreover, one can consider specific protein mass or molecule count per cell, thus reflecting changes in cell size across conditions.
Our analysis focuses on the relations between different proteins and resource distribution inside the proteome, and thus avoids such metrics.

\paragraph{Filtering out conditions from the data set from Schmidt et. al.}


The \cite{Schmidt2015} data set contains proteomic data measurements under 22 different environmental conditions.
However, our model assumes exponential growth, implying that measurements taken at stationary phase are expected to differ from simple extrapolation of the model to zero growth rate.
Therefore, the two measurements of stationary phase proteomics were excluded from our analysis.

Out of the conditions measured in the \cite{Schmidt2015} data set, two conditions included amino acids in the media and presented much faster growth rate than the rest of the conditions (growth in LB media and in glycerol supplemented with AA, with growth rates of $1.9[h^{-1}]$ and $1.27[h^{-1}]$ respectively, compared with a range of $0.12-0.66$ for the other conditions).
This asymmetry in the distribution of growth rates caused inclusion of these conditions to dominate the analysis due to its effect on the skewness of the distribution of growth rates ($\gamma_1=-0.5$ for the growth rates excluding LB and AA supplemented glycerol vs. $\gamma_1=2.3$ with LB and AA supplemented glycerol) reducing the statistical power of the other conditions.
While including the data on growth in these conditions does not qualitatively change the observed results, such analysis is much less statistically robust.
We have therefore omitted growth in LB and in AA supplemented glycerol in the main analysis.
We present the analysis including these conditions in \nameref{lbanalysis}.

\section*{Acknowledgments}
We kindly thank Matthias Heinemann for making proteomics data available before publication.
We thank Uri Alon, Stephan Klumpp, Arren Bar Even, Avi Flamholz, Elad Noor, Kaspar Valgepea, Karl Peebo, Dan Davidi, Niv Antonovsky, Yinon Bar-On and Katja Tummler for fruitful discussions and enlightening comments on this work.

\begin{thebibliography}{10}

\bibitem{Schaechter1958}
Schaechter M, Maaloe O, Kjeldgaard NO.
\newblock Dependency on Medium and Temperature of Cell Size and Chemical
  Composition during Balanced Growth of Salmonella typhimurium.
\newblock Microbiology. 1958;19(3):592--606.
\newblock Available from:
  \url{http://mic.microbiologyresearch.org/content/journal/micro/10.1099/00221287-19-3-592}.

\bibitem{Neidhardt1999}
Neidhardt FC.
\newblock {Bacterial Growth: Constant Obsession with dN/dt}.
\newblock J Bacteriol. 1999 dec;181(24):7405--7408.
\newblock Available from: \url{http://jb.asm.org}.

\bibitem{Dennis2004}
Dennis PP, Ehrenberg M, Bremer H.
\newblock Control of {rRNA} synthesis in Escherichia coli: a systems biology
  approach.
\newblock Microbiol Mol Biol Rev. 2004 Dec;68(4):639--668.

\bibitem{Zaslaver2009}
Zaslaver A, Kaplan S, Bren A, Jinich A, Mayo A, Dekel E, et~al.
\newblock {Invariant distribution of promoter activities in Escherichia coli.}
\newblock PLoS computational biology. 2009 oct;5(10):e1000545.
\newblock Available from:
  \url{http://dx.plos.org/10.1371/journal.pcbi.1000545}.

\bibitem{Molenaar2009}
Molenaar D, van Berlo R, de~Ridder D, Teusink B.
\newblock {Shifts in growth strategies reflect tradeoffs in cellular
  economics.}
\newblock Molecular systems biology. 2009 jan;5:323.
\newblock Available from: \url{http://dx.doi.org/10.1038/msb.2009.82
  http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2795476\&tool=pmcentrez\&rendertype=abstract}.

\bibitem{Nomura1984}
Nomura M, Gourse R, Baughman G.
\newblock Regulation of the synthesis of ribosomes and ribosomal components.
\newblock Annual review of biochemistry. 1984;53:75--117.

\bibitem{Murray_2003}
Murray HD, Schneider DA, Gourse RL.
\newblock {Control of {rRNA} Expression by Small Molecules Is Dynamic and
  Nonredundant}.
\newblock Molecular Cell. 2003 jul;12(1):125--134.
\newblock Available from:
  \url{http://dx.doi.org/10.1016/s1097-2765(03)00266-1}.

\bibitem{Bosdriesz_2015}
Bosdriesz E, Molenaar D, Teusink B, Bruggeman FJ.
\newblock {How fast-growing bacteria robustly tune their ribosome concentration
  to approximate growth-rate maximization}.
\newblock {FEBS} Journal. 2015 mar;Available from:
  \url{http://dx.doi.org/10.1111/febs.13258}.

\bibitem{Chatterji2001}
Chatterji D, Ojha AK.
\newblock Revisiting the stringent response, ppGpp and starvation signaling.
\newblock Current Opinion in Microbiology. 2001 Apr;4(2):160--165.

\bibitem{Brauer2008}
Brauer MJ, Huttenhower C, Airoldi EM, Rosenstein R, Matese JC, Gresham D,
  et~al.
\newblock {Coordination of growth rate, cell cycle, stress response, and
  metabolic activity in yeast.}
\newblock Molecular biology of the cell. 2008 jan;19(1):352--67.
\newblock Available from:
  \url{http://www.molbiolcell.org/cgi/content/abstract/19/1/352}.

\bibitem{Saldanha2004}
Saldanha AJ, Brauer MJ, Botstein D.
\newblock {Nutritional homeostasis in batch and steady-state culture of yeast.}
\newblock Molecular biology of the cell. 2004;15(9):4089--4104.

\bibitem{You_2013}
You C, Okano H, Hui S, Zhang Z, Kim M, Gunderson CW, et~al.
\newblock {Coordination of bacterial proteome with metabolism by cyclic {AMP}
  signalling}.
\newblock Nature. 2013 aug;500(7462):301--306.
\newblock Available from: \url{http://dx.doi.org/10.1038/nature12446}.

\bibitem{Peebo_2015}
Peebo K, Valgepea K, Maser A, Nahku R, Adamberg K, Vilu R.
\newblock {Proteome reallocation in Escherichia coli with increasing specific
  growth rate}.
\newblock Mol {BioSyst}. 2015;11(4):1184--1193.
\newblock Available from: \url{http://dx.doi.org/10.1039/c4mb00721b}.

\bibitem{Keren2013}
Keren L, Zackay O, Lotan-Pompan M, Barenholz U, Dekel E, Sasson V, et~al.
\newblock {Promoters maintain their relative activity levels under different
  growth conditions}.
\newblock Molecular Systems Biology. 2013;9:701.
\newblock Available from:
  \url{http://www.nature.com/doifinder/10.1038/msb.2013.59}.

\bibitem{Castrillo2007}
Castrillo JI, Zeef LA, Hoyle DC, Zhang N, Hayes A, Gardner DCJ, et~al.
\newblock {Growth control of the eukaryote cell: a systems biology study in
  yeast.}
\newblock Journal of biology. 2007;6(2):4.

\bibitem{Gerosa2013}
Gerosa L, Kochanowski K, Heinemann M, Sauer U.
\newblock {Dissecting specific and global transcriptional regulation of
  bacterial gene expression.}
\newblock Molecular systems biology. 2013;9(658):658.
\newblock Available from:
  \url{http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=3658269\&tool=pmcentrez\&rendertype=abstract}.

\bibitem{Gasch2000}
Gasch AP, Spellman PT, Kao CM, Carmel-Harel O, Eisen MB, Storz G, et~al.
\newblock {Genomic expression programs in the response of yeast cells to
  environmental changes.}
\newblock Molecular biology of the cell. 2000;11(12):4241--4257.

\bibitem{Klumpp2009}
Klumpp S, Zhang Z, Hwa T.
\newblock {Growth rate-dependent global effects on gene expression in
  bacteria.}
\newblock Cell. 2009 dec;139(7):1366--75.
\newblock Available from: \url{http://dx.doi.org/10.1016/j.cell.2009.12.001}.

\bibitem{Klumpp2014}
Klumpp S, Hwa T.
\newblock {Bacterial growth: global effects on gene expression, growth feedback
  and proteome partition}.
\newblock Current Opinion in Biotechnology. 2014;28:96--102.
\newblock Available from:
  \url{http://linkinghub.elsevier.com/retrieve/pii/S0958166914000044}.

\bibitem{Scott2010}
Scott M, Gunderson CW, Mateescu EM, Zhang Z, Hwa T.
\newblock {Interdependence of cell growth and gene expression: origins and
  consequences.}
\newblock Science (New York, NY). 2010 nov;330(6007):1099--102.
\newblock Available from: \url{http://www.ncbi.nlm.nih.gov/pubmed/21097934}.

\bibitem{Scott2011}
Scott M, Hwa T.
\newblock {Bacterial growth laws and their applications.}
\newblock Current opinion in biotechnology. 2011 aug;22(4):559--65.
\newblock Available from:
  \url{http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=3152618\&tool=pmcentrez\&rendertype=abstract}.

\bibitem{Scott2014}
Scott M, Klumpp S, Mateescu EM, Hwa T.
\newblock {Emergence of robust growth laws from optimal regulation of ribosome
  synthesis.}
\newblock Molecular systems biology. 2014 jan;10(8):747.
\newblock Available from: \url{http://www.ncbi.nlm.nih.gov/pubmed/25149558}.

\bibitem{Berthoumieux2013}
Berthoumieux S, de~Jong H, Baptist G, Pinel C, Ranquet C, Ropers D, et~al.
\newblock {Shared control of gene expression in bacteria by transcription
  factors and global physiology of the cell.}
\newblock Molecular systems biology. 2013 jan;9(634):634.
\newblock Available from:
  \url{http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=3564261\&tool=pmcentrez\&rendertype=abstract}.

\bibitem{Valgepea2013}
Valgepea K, Adamberg K, Seiman A, Vilu R.
\newblock {Escherichia coli achieves faster growth by increasing catalytic and
  translation rates of proteins.}
\newblock Molecular BioSystems. 2013 sep;9(9):2344--58.

\bibitem{Hui_2015}
Hui S, Silverman JM, Chen SS, Erickson DW, Basan M, Wang J, et~al.
\newblock {Quantitative proteomic analysis reveals a simple strategy of global
  resource allocation in bacteria}.
\newblock Molecular Systems Biology. 2015 feb;11(2):e784--e784.
\newblock Available from: \url{http://dx.doi.org/10.15252/msb.20145697}.

\bibitem{Weise2015}
Wei{\ss}e AY, Oyarz{\'{u}}n DA, Danos V, Swain PS.
\newblock {Mechanistic links between cellular trade-offs, gene expression, and
  growth.}
\newblock Proceedings of the National Academy of Sciences of the United States
  of America. 2015 feb;112(9):E1038--1047.
\newblock Available from:
  \url{http://www.pnas.org/content/112/9/E1038.abstract}.

\bibitem{Klumpp2008}
Klumpp S, Hwa T.
\newblock {Growth-rate-dependent partitioning of RNA polymerases in bacteria.}
\newblock Proceedings of the National Academy of Sciences of the United States
  of America. 2008 dec;105(51):20245--50.
\newblock Available from:
  \url{http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2629260\&tool=pmcentrez\&rendertype=abstract}.

\bibitem{Regenberg2006}
Regenberg B, Grotkjaer T, Winther O, Fausb\o~ll A, Akesson M, Bro C, et~al.
\newblock {Growth-rate regulated genes have profound impact on interpretation
  of transcriptome profiling in Saccharomyces cerevisiae.}
\newblock Genome biology. 2006;7(11):R107.

\bibitem{Schmidt2015}
Schmidt A, Kochanowski K, Vedelaar S, Ahrne E, Volkmer B, Callipo L, et~al.
\newblock {The quantitative and condition-dependent Escherichia coli proteome}.
\newblock Nat Biotech. 2015 dec;advance on.
\newblock Available from: \url{http://dx.doi.org/10.1038/nbt.3418
  10.1038/nbt.3418
  http://www.nature.com/nbt/journal/vaop/ncurrent/abs/nbt.3418.html{\#}supplementary-information}.

\bibitem{Maaloe1969}
Maaloe O.
\newblock {An analysis of bacterial growth}.
\newblock Dev Biol Suppl. 1969;3:33--58.
\newblock Available from:
  \url{http://scholar.google.com/scholar?hl=en\&btnG=Search\&q=intitle:An+analysis+of+bacterial+growth\#0}.

\bibitem{Bremer1987}
Bremer H, Dennis PP.
\newblock {Modulation of chemical composition and other parameters of the cell
  by growth rate}.
\newblock In: Neidhart FC, editor. Escherichia coli and Salmonella: Cellular
  and Molecular Biology. vol.~2. 2nd ed. Washington D.C.: ASM Press; 1996. p.
  1553--1569.
\newblock Available from:
  \url{https://www.researchgate.net/profile/Patrick_Dennis/publication/237130769_Modulation_of_Chemical_Composition_and_Other_Parameters_of_the_Cell_by_Growth_Rate/links/0c96053c704c10d62e000000.pdf}.

\bibitem{24656150}
Li Z, Nimtz M, Rinas U.
\newblock {The metabolic potential of Escherichia coli BL21 in defined and rich
  medium.}
\newblock Microb Cell Fact. 2014 Mar;13:45.

\bibitem{10515934}
Tao H, Bausch C, Richmond C, Blattner F, Conway T.
\newblock {Functional genomics: expression analysis of Escherichia coli growing
  on minimal and rich media.}
\newblock J Bacteriol;181:6425--40.

\bibitem{ingraham1983growth}
Ingraham JL, Maal\o~e O, Neidhardt FC.
\newblock {Growth of the bacterial cell}.
\newblock Sinauer Associates; 1983.
\newblock Available from:
  \url{http://books.google.co.il/books?id=6vZqAAAAMAAJ}.

\bibitem{Pedersen1978a}
Pedersen S, Bloch PL, Reeh S, Neidhardt FC.
\newblock {Patterns of protein synthesis in E. coli: a catalog of the amount of
  140 individual proteins at different growth rates.}
\newblock Cell. 1978;14(1):179--190.
\newblock Available from: \url{http://www.ncbi.nlm.nih.gov/pubmed/352533}.

\bibitem{Christiano2014}
Christiano R, Nagaraj N, Fr{\"{o}}hlich F, Walther TC.
\newblock {Global Proteome Turnover Analyses of the Yeasts S. cerevisiae and S.
  pombe.}
\newblock Cell reports. 2014;p. 1959--1965.
\newblock Available from:
  \url{http://linkinghub.elsevier.com/retrieve/pii/S2211124714009346$\backslash$nhttp://www.ncbi.nlm.nih.gov/pubmed/25466257}.

\bibitem{Paalme_1995}
Paalme T, Kahru A, Elken R, Vanatalu K, Tiisma K, Raivo V.
\newblock {The computer-controlled continuous culture of Escherichia coli with
  smooth change of dilution rate (A-stat)}.
\newblock Journal of Microbiological Methods. 1995 dec;24(2):145--153.
\newblock Available from: \url{http://dx.doi.org/10.1016/0167-7012(95)00064-x}.

\bibitem{23203884}
Salgado H, Peralta-Gil M, Gama-Castro S, Santos-Zavaleta A, Mu{\~{n}}iz-Rascado
  L, Garc{\'{\i}}a-Sotelo J, et~al.
\newblock {RegulonDB v8.0: omics data sets, evolutionary conservation,
  regulatory phrases, cross-validated gold standards and more.}
\newblock Nucleic Acids Res;41:D203--13.

\bibitem{Kaneko2014}
Kaneko K, Furusawa C, Yomo T.
\newblock {Universal relationship in gene-expression changes for cells in
  steady-growth state}. 2014 jul;p.~7.
\newblock Available from: \url{http://arxiv.org/abs/1407.3622}.

\bibitem{Srivatsan2008}
Srivatsan A, Wang JD.
\newblock {Control of bacterial transcription, translation and replication by
  (p)ppGpp.}
\newblock Current opinion in microbiology. 2008;11(2):100--5.
\newblock Available from: \url{http://www.ncbi.nlm.nih.gov/pubmed/18359660}.

\bibitem{Oliphant2007}
Oliphant TE.
\newblock {SciPy: Open source scientific tools for Python}.
\newblock Computing in Science and Engineering. 2007;9:10--20.
\newblock Available from: \url{http://www.scipy.org/}.

\bibitem{Community2011}
Community N.
\newblock {NumPy Reference}.
\newblock October. 2011;1(October):1--1146.

\bibitem{McKinney2011}
McKinney W.
\newblock {pandas: a Foundational Python Library for Data Analysis and
  Statistics}.
\newblock In: Python for High Performance and Scientific Computing; 2011. p.
  1--9.

\bibitem{Hunter2007}
Hunter JD.
\newblock {Matplotlib: A 2D Graphics Environment}.
\newblock Computing in Science Engineering. 2007;9(3):90--95.
\newblock Available from:
  \url{http://ieeexplore.ieee.org/lpdocs/epic03/wrapper.htm?arnumber=4160265}.

\end{thebibliography}

\section*{Supporting Information}
\clearpage
\subsection*{S1 Text}
\label{theoreticalintercep}
\subsection*{S1 Fig}
\label{fig:theoreticalpred}
{\bf The predictions of the model for the fraction of unregulated proteins as a function of the growth rate.}
The effects of accounting for protein degradation (green) and Michaelis-Menten like dependence of bio-synthesis rates on growth rate (red) are shown.
For non-constant bio-synthesis rate, a growth rate of 0.2 was selected as the growth rate at which the bio-synthesis rate is half of its maximal value.%
\subsection*{S2 Text}
\label{furtherdatasetanalysis}
\subsection*{S2 Fig}
\label{fig:grcorr_dbs}
    {\bf Histograms of the correlation with growth rate of proteins in data sets obtained by gradually modifying the growth rate with constant input media.}
All such data sets present similar characteristics with very high correlation or anti correlation of protein fraction out of the proteome with the growth rate.
Such data sets make it hard to distinguish between specific regulation mechanisms and passive effects and are therefore inadequate for testing the passive resource allocation model.
(A) 4 chemostat growth conditions from \cite{Schmidt2015}.
(B) 9 glucose accelerostat conditions from \cite{Peebo_2015}.
(C) 5 glucose accelerostat conditions from \cite{Valgepea2013}.
(D-F) different growth rate modification methods from \cite{Hui_2015}.
\subsection*{S3 Text}
\label{dbscorr}
\subsection*{S3 Fig}
\label{fig:chemcorrcomp}
    {\bf Pair-wise comparison of the data sets used in this work.}
Each panel compares the correlation with growth rate of all proteins between two data sets.
The conditions used from each data set utilize similar growth conditions, except for the data sets from \cite{Hui_2015} (see text).
\subsection*{S4 Text}
\label{corrthreshold}
\subsection*{S4 Fig}
\label{fig:threshold}
    {\bf Statistics on the explained variability in the normalized data set as a function of the threshold used for defining strong correlation with growth rate.}
  An optimal threshold is a threshold that maximizes the fraction of explained variability in the proteome by linear regression on proteins that have a correlation with growth rate that exceeds the threshold (blue line).
  The maximal fraction of explained variability in each data set is marked as a horizontal dashed line and is $\hMaxExpVar{}$ (obtained given a threshold of $0.25$) for the data set from \cite{Schmidt2015}, and $\vMaxExpVar{}$ (obtained given a threshold of $0.2$) in the data from \cite{Peebo_2015}.
\subsection*{S5 Text}
\label{slopedistexp}
\subsection*{S6 Text}
\label{lbanalysis}
\subsection*{S5 Fig}
\label{fig:LB}
    {\bf Including growth in LB and AA supplemented glycerol media in the analysis of the data set from \cite{Schmidt2015}.}
Fewer proteins are strongly positively correlated with growth but these proteins form more than $50\%$ of the proteome in fast growth.%
\subsection*{S7 Text}
\label{ribosomeconc}
\subsection*{S6 Fig}
\label{fig:randpred}
    {\bf The fraction out of the proteome of proteins that are not differentially regulated between conditions can be predicted by referencing other such proteins.}
  A selection of random predictions of protein fractions from the highly correlated with growth rate group, taken from the data set of \cite{Schmidt2015}.
  Each panel shows the average fraction of 10 random proteins that are highly correlated with growth (blue dots), a regression line that best fits the data, and the fraction of a different random protein (green dots).
  The $R^2$ value for the trend line and the different protein is given.
\subsection*{S1 Table}
\bf Breakdown by function of strongly positively correlated with growth rate proteins in the data set from \cite{Schmidt2015}.
\label{tab:corrbreakdownh}
\subsection*{S2 Table}
\bf Breakdown by function of strongly positively correlated with growth rate proteins in the data set from \cite{Peebo_2015}.
\label{tab:corrbreakdownv}
\subsection*{S7 Fig}
\label{fig:ribsnonribs}
    {\bf Ribosomal proteins scale similarly to non-ribosomal proteins that are strongly positively correlated with growth rate.}
 The scaling with growth rate of ribosomal proteins and non-ribosomal, but highly correlated with growth rate proteins is shown.
Comparing the normalized sum of ribosomal proteins to the normalized sum of the positively correlated with growth rate proteins that are non-ribosomal shows that these two groups scale in a similar way with the growth rate.
Trend lines for the two groups of proteins are plotted.
\subsection*{S8 Text}
\label{scottcompare}
\subsection*{S8 Fig}
\label{fig:shuffledexpvar}
    {\bf Fraction of explained variability by linear regression on the group of strongly positively correlated with growth rate proteins for the shuffled data sets.}
The maximal explained variability in these data sets is significantly smaller than in the real data sets
\subsection*{S9 Fig}
\label{fig:simulated}
    {\bf A simulated data set, assuming half of the proteins are perfectly correlated with growth rate and half are fixed, with simulated noise level of $25\%$.}
Average protein fractions, growth rates and normalized slope of the correlated proteins are based on the data set from \cite{Schmidt2015}.
The normalized intercept of the correlated proteins was set to $0.5$ in accordance with the intercept found in the original data analysis.
The results are similar to those obtained for the real data set, showing that, given the experimental noise, identical coordination with growth rate of half of the proteins would result in similar outcomes to those observed in the data sets we use.%

\end{document}

